### PR TITLE
Add additional mappings for name_suffixes

### PR DESCRIPTION
**Pull request by money**
### Jira Story

- [Inconsistent results for suffixes INT-1663](https://osi-cwds.atlassian.net/browse/INT-1663)

## Description
Inconsistent results for I, II, III, IV suffixes in Relationship and History, Suffixes shows as numbers (1,2,3,4) instead because of how the data is sent by api.  Added the missing mapping to help fix the bug.

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 
Bug fix.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/utils/nameFormatter.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import NAME_SUFFIXES from 'enums/NameSuffixes'
 
 const addSuffix = (name, suffix) => {
-  if (['ii', 'iii', 'iv'].includes(suffix)) {
+  if (['ii', 'iii', 'iv', '2', '3', '4'].includes(suffix)) {
```
Comment (wmitchell-cambria) [inline app/javascript/utils/nameFormatter.js]: Would something like `Object.keys(NAME_SUFFIXES)` work here to allow us to avoid duplicating the list?

**Comment by NagaPradeepGajula-osi** [comment]
Thanks for including the changes.

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

File: app/javascript/enums/NumberSuffixes.js [hunk 2]
```diff
@@ -97,6 +97,55 @@ const NUMBER_SUFFIXES = Object.freeze({
...
+  XLI: 'XLI',
+  XLII: 'XLII',
+  XLIII: 'XLIII',
+  XLIV: 'XLIV',
+  XLV: 'XLV',
+  XLVI: 'XLVI',
+  XLVII: 'XLVII',
+  XLVIII: 'XLVIII',
+  XLIX: 'XLIX',
+  L: 'L',
```
Comment (wmitchell-cambria) [inline app/javascript/enums/NumberSuffixes.js]: Money had mentioned to me that we expect all suffixes from the API to be lower case. Are we getting capitalized suffixes from somewhere else?

Comment (NagaPradeepGajula-osi) [reply app/javascript/enums/NumberSuffixes.js]: yes we found in case of one mock data person.

Comment (TylerOSI) [reply app/javascript/enums/NumberSuffixes.js]: there no need to duplicate this mapping, just lowercase the key before doing the lookup

**Pull request merged**